packages/components/inputs/search-text-input/README.md

```diff
@@ -60,6 +60,7 @@ export default Example;
 | `onSubmit`             | `Function`<br/>[See signature.](#signature-onSubmit)                                                  |    ✅    |           | Handler when the search button is clicked.                                                                                |
 | `onReset`              | `Function`<br/>[See signature.](#signature-onReset)                                                   |    ✅    |           | Handler when the clear button is clicked.                                                                                 |
 | `isAutofocussed`       | `boolean`                                                                                             |          |           | Focus the input on initial render                                                                                         |
+| `isCondensed`          | `boolean`                                                                                             |          |           | Whether the options appear in a condensed layout                                                                          |
```

Comment:
You need to generate the Readme again to get the latest JSDoc comment from the propw type.

Reply:
Thanks!